README.md

```diff
@@ -35,10 +35,15 @@ Add to Android manifest the next permission:
 
 Declare `DeviceAnimationTestRule` as an static field annotated with `@ClassRule` to your suit: 
 
+Java:
 ```java
-@ClassRule static public DeviceAnimationTestRule
-      deviceAnimationTestRule = new DeviceAnimationTestRule();
+@ClassRule static public DeviceAnimationTestRule deviceAnimationTestRule = new DeviceAnimationTestRule();
+```
+or Kotlin:
+```kotlin
+ companion object {
+    @ClassRule
+    @JvmField
```

Comment:
I think you can use `@get:ClassRule` here instead.

Reply:
To run Rule once for class you should use the @ClassRule and @JvmField instead of @get:ClassRule. 

@get:ClassRule runs before each test